.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# Set update schedule for GitHub Actions
```

Comment:
I think that this is meant to be in `.github/dependabot.yml`

Reply:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#example-dependabotyml-file-for-github-actions i think i misread this, will move it